index.php

```diff
@@ -141,5 +141,15 @@
     </script><noscript><p><img src="<?php echo $piwikConf["url"]; ?>piwik.php?idsite=<?php echo $piwikConf["siteId"]; ?>" style="border:0" alt="" /></p></noscript>
 <?php } ?>
 <!-- End Piwik Tracking Code -->
+
+<!-- temporary tracking of page views -->
```

Comment:
At first I was confused because the previous lines also add Piwik tracking. I'd rather call this `temporary tracking of page views with donation tracker` or something similar